packages/block-library/src/cover/block.json

```diff
@@ -28,6 +28,9 @@
 		},
 		"focalPoint": {
 			"type": "object"
+		},
+		"height": {
+			"type": "number"
```

Comment:
Can we add the default height here? `"default": 430`

Reply:
Adding the default here will invalidate previous blocks. We'll need a deprecation if we do so.